tasks/eva_3158/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM python:3.10
+
+ENV vcf_validator_version=0.9.4
+ENV python_version=3.10.9
+ENV submission_automation_version=v1.9.0
+ENV samtools_version=1.16.1
+ENV bcfools_version=1.16
+ENV htslib_version=1.16
+ENV NXF_VER=22.10.6
+
+WORKDIR /opt
+
+RUN mkdir vcf_validator \
+    && cd vcf_validator  \
+    && curl -LJo vcf_validator  https://github.com/EBIvariation/vcf-validator/releases/download/v${vcf_validator_version}/vcf_validator_linux \
+    && curl -LJo vcf_assembly_checker  https://github.com/EBIvariation/vcf-validator/releases/download/v${vcf_validator_version}/vcf_assembly_checker_linux \
+    && chmod 755 vcf_assembly_checker vcf_validator \
+    && cd ..
+
+# Install python
+#RUN curl -LJO https://www.python.org/ftp/python/${python_version}/Python-${python_version}.tgz
+#RUN tar -zxf Python-${python_version}.tgz
+#RUN rm Python-${python_version}.tgz
+#RUN cd Python-${python_version}  \
+#    && ./configure --prefix=$(pwd) > configure.log
+#RUN cd Python-3.10.9 && make > make.log 2>&1
+#RUN cd Python-3.10.9 && make altinstall > install.log 2>&1
+#RUN cd ..
+#RUN Python-${python_version}/python -m venv ./venv
```

Comment:
I guess you don't need these as you're using a python image - maybe you're just keeping them to reference in the future?

Reply:
Yes that's right. I attempted to use a bare bone image but it turned out to be too time consuming so reverted to the fully fledge image to get something to work. 
Just leaving this here for future reference.